psi4/driver/procrouting/proc.py

```diff
@@ -1272,6 +1272,29 @@ def scf_helper(name, post_scf=True, **kwargs):
     if ref_wfn is not None:
         raise ValidationError("Cannot seed an SCF calculation with a reference wavefunction ('ref_wfn' kwarg).")
 
+    # PCM needs to be run w/o symmetry
```

Comment:
Could you add a warning or print message that these options have been set under the hood?